### DISCUSSION TITLE
1.0

### DISCUSSION THREAD

**Discussion by KhatimaSadat** [Afghn-Girls-Tech-Team/Web-development#10] [Announcements]
[README.md](https://github.com/Afghn-Girls-Tech-Team/Web-development/files/8839809/README.md)

<hr /><em>This discussion was created from the release <a href='https://github.com/Afghn-Girls-Tech-Team/Web-development/releases/tag/'>1.0</a>.</em>

**Comment by KhatimaSadat** [comment]
i have added http server to our project what is your idea about this plz say something @TahiraMirzaie @Nooriaehsani @ShaqayeqRahmani @Parvanah

**Reply by Parvanah** [reply]
this is a good idea for the betterment of the project.

**Reply by TahiraMirzaie** [reply]
This is a good for our project

**Reply by ShaqayeqRahmani** [reply]
we should add,it was necessary BTW good

**Reply by Nooriaehsani** [reply]
This is better that it add
